README.md

```diff
@@ -2,11 +2,19 @@
 
 ---
 
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-52-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
-
-[![Netlify Status](https://api.netlify.com/api/v1/badges/b2137407-b765-46c4-95b5-a72d9b1592ab/deploy-status)](https://app.netlify.com/sites/asyncapi-website/deploys)
+<br/>
+<p align="center">
+<a href="https://github.com/asyncapi/website" alt="AsyncAPI GitHub website contributors">
+<img src="https://img.shields.io/github/contributors/asyncapi/website.svg?color=orange" /></a>
```

Comment:
Why contributors are shown as 169? on what logic?

Reply:
I noticed `https://img.shields.io/github/contributors` doesn't show exact contributor numbers.

like `https://img.shields.io/github/contributors/asyncapi/website` shows 169 but it is 171
`https://img.shields.io/github/contributors/asyncapi/asyncapi-react` shows 31 but it is 34